File: lambda/hello_world.js

exports.helloWorld = async function (event, context) {
    try {
        var method = event.httpMethod;

        if (method === "GET") {
            if (event.path === "/api/") {
                return {
                    statusCode: 200,
                    headers: {},
                    body: JSON.stringify("Hello World")
                };
            }
        }

        // We only accept GET for now
        return {
            statusCode: 400,
            headers: {},
            body: JSON.stringify("We only accept GET /api/ - got " + method + " " + event.path)
        };
    } catch (error) {
        var body = error.stack || JSON.stringify(error, null, 2);
        return {
            statusCode: 400,
            headers: {},
            body: JSON.stringify(body)
        }
    }
}

File: lambda/hello_world.py
import json
import os
import boto3

TABLE_NAME = os.environ['TABLE_NAME']
PRIMARY_KEY = os.environ['PRIMARY_KEY'] 

def helloWorld(event, context): 
    dynamodb = boto3.client('dynamodb')
    element_count = dynamodb.describe_table(TableName=TABLE_NAME)['Table']['ItemCount']
    return {
        'statusCode': 200,
        'body': json.dumps(str(element_count) + " elements in the table."),
        'headers' : {
            'Cache-Control': 'no-cache'
        }
    }